e2e/e2etests/e2etests.go

```diff
@@ -61,7 +61,7 @@ const (
 	TestSolanaDepositAndCallRefundName = "solana_deposit_and_call_refund"
 	TestSolanaDepositRestrictedName    = "solana_deposit_restricted"
 	TestSolanaWithdrawRestrictedName   = "solana_withdraw_restricted"
-
+	TestSolanaWhitelistSPLName         = "solana_whitelist_spl"
```

Comment:
It's testing admin functions so would be consistent to have it in the admin test suite.
> that seems like a bit of an overkill because it should set up solana program, and everything solana related,
I would think that this is fine. But might be out of scope for this PR so we could create an issue